CMakeLists.txt

```diff
@@ -122,16 +122,30 @@ set_local_or_system_option(
     "GTEST" OFF
     "Build the GoogleTest framework by providing your own copy of the repo in ext/googletest (see Local Builds in README)"
 )
+option(AVIF_BUILD_APPS "Build avif apps." OFF)
+option(AVIF_BUILD_TESTS "Build avif tests." OFF)
 option(AVIF_LOCAL_FUZZTEST
        "Build the Google FuzzTest framework by providing your own copy of the repo in ext/fuzztest (see Local Builds in README)"
        OFF
 )
-
 option(
     AVIF_ENABLE_COMPLIANCE_WARDEN
     "Check all avifEncoderFinish() output for AVIF specification compliance. Depends on gpac/ComplianceWarden which can be added with ext/compliance_warden.sh"
     OFF
 )
+option(
+    AVIF_ENABLE_GOLDEN_TESTS
+    "Build tests that compare encoding outputs to golden files. Needs AVIF_BUILD_APPS=ON, and depends on MP4box which can be built with ext/mp4box.sh"
+    OFF
+)
+option(AVIF_ENABLE_GTEST
```

Comment:
Optional: (Can do this in a separate CL.) Should we declare the `AVIF_ENABLE_GTEST` and `AVIF_ENABLE_FUZZTEST` options with `cmake_dependent_option()` and make them depend on `AVIF_BUILD_TESTS`?

Reply:
that is probably cleaner indeed, though the doc is right here at least (compare to `AVIF_ENABLE_GOLDEN_TESTS`)